src/main/java/com/notably/commons/core/path/AbsolutePath.java

```diff
@@ -1,53 +1,113 @@
 package com.notably.commons.core.path;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.Objects;
 
 import com.notably.commons.core.path.exceptions.InvalidPathException;
 
 /**
  * TODO: Add Javadoc
  */
 public class AbsolutePath implements Path {
+    public static final String INVALIDABSOLUTEPATH = "Invalid absolute path";
+    private final List<String> paths;
+
+
+    public AbsolutePath(String absolutePathString) {
+        this.paths = new ArrayList<>();
+        for (String obj : absolutePathString.split("/")) {
+            if (!obj.trim().equals("")) {
+                this.paths.add(obj.trim());
+            }
+        }
+    }
+
     /**
-     * TODO: Add Javadoc
+     * Check if string is a ValidAbsolutePath
+     * @param absolutePathString
+     * @return
      */
     public static boolean isValidAbsolutePath(String absolutePathString) {
-        // TODO: Add implementation
+        if (absolutePathString.charAt(0) == '/') {
+            if (absolutePathString.contains("../")) {
+                return false;
+            } else {
+                return true;
+            }
+        }
         return false;
     }
 
     /**
-     * TODO: Add Javadoc
+     * Creates an absolute path from a string.
+     * @param absolutePathString used to create an absolutePath.
+     * @return Absolute Path.
+     * @throws InvalidPathException if String provided is not a valid absolutePath.
      */
     public static AbsolutePath fromString(String absolutePathString) throws InvalidPathException {
-        // TODO: Add implementation
-        return new AbsolutePath();
+        if (isValidAbsolutePath(absolutePathString)) {
+            return new AbsolutePath(absolutePathString);
+        } else {
+            throw new InvalidPathException(INVALIDABSOLUTEPATH);
```

Comment:
Just FYI: You don't actually need the `else` branch here. If the `if` branch is true, the program would have returned. For that reason, you can rewrite this as

```java
if (isValidAbsolutePath(absolutePathString)) {
  return new AbsolutePath(absolutePathString);
}
throw new InvalidPathException(INVALID_ABSOLUTE_PATH);
```

However, even better, we can structure this condition following the [happy path](https://en.wikipedia.org/wiki/Happy_path) principle, as such

```java
if (!isValidAbsolutePath(absolutePathString)) {
  throw new InvalidPathException(INVALID_ABSOLUTE_PATH);
}
return new AbsolutePath(absolutePathString);